### ISSUE TITLE
get_group for groupby

### ISSUE THREAD

**Issue by reza1615** [man-group/dtale#134]
It is interesting to have an option to get_group
Use case:

https://stackoverflow.com/questions/14734533/how-to-access-pandas-groupby-dataframe-by-key

**Comment by aschonfeld** [comment]
So essentially you want to be able to extract a specific group into a new data window?

**Comment by reza1615** [comment]
Yes it would be intresting to have extract specific group into a new instance
